static/css/nl_interface.scss

```diff
@@ -48,19 +49,19 @@ $answer-min-height: 100vh;
 
 #results-thread-container {
   padding: 0;
-  margin-bottom: 7rem; // approx height of #search-container
+  margin-bottom: $search-section-bottom-height;
 }
 
 #search-section-container-center {
-  position: fixed;
-  bottom: 50%;
-  transform: translate(0, 50%);
+  position: absolute;
+  top: calc(50% - #{$search-section-bottom-height});
 }
 
 #search-section-container-bottom {
   position: fixed;
-  bottom: 0;
-  transition: bottom 225ms cubic-bezier(0, 0, 0.2, 1) 0ms;
+  top: calc(100% - #{$search-section-bottom-height});
```

Comment:
does it still work to set bottom (since there isn't anything else setting it's top anymore). setting `bottom: 0` is much cleaner than the calculations

Reply:
setting bottom won't work with the transition because bottom won't be set in the original state ..